# HG changeset patch
# User Rik <rik@octave.org>
# Date 1383768141 28800
#      Wed Nov 06 12:02:21 2013 -0800
# Node ID 46850674da463056c098b6cc351b592998b1760b
# Parent  870f3e12e16322ac290a9b20e23b6df80931d86b
NEWS: Add sentence about octave_core_file_name function.

* NEWS: Add sentence about octave_core_file_name function.

diff --git a/NEWS b/NEWS
--- a/NEWS
+++ b/NEWS
@@ -264,20 +264,22 @@ Summary of important user-visible change
  ** The glpk function has been modified to reflect changes in the GLPK
     library.  The "round" and "itcnt" options have been removed.  The
     "relax" option has been replaced by the "rtest" option.  The numeric
     values of error codes and of some options have also changed.
 
  ** The kurtosis function has changed definition to be compatible with 
     Matlab.  It now returns the base kurtosis instead of the "excess kurtosis".
     The old behavior can be had by changing scripts to normalize with -3.
+
                "excess kurtosis" = kurtosis (x) - 3
 
  ** The default name of the Octave crash dump file is now 
-    "octave-workspace" instead of "octave-core".
+    "octave-workspace" instead of "octave-core".  The exact name can
+    always be customized with the octave_core_file_name function.
 
  ** A citation command has been added to display information on how to
     cite Octave and packages in publications.  The package system will
     look for and install CITATION files from packages.
 
  ** The java package from Octave Forge is now part of core Octave.  The
     following new functions are available for interacting with Java
     directly from Octave:
